source/code-snippets/authentication/aws.js

```diff
@@ -0,0 +1,32 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with values for your environment.
+const accessKeyId = encodeURIComponent("<AWS_ACCESS_KEY_ID>");
+const secretAccessKey = encodeURIComponent("<AWS_SECRET_ACCESS_KEY>");
+const clusterUrl = "<MongoDB cluster url>";
+
+const authMechanism = "MONGODB-AWS";
```

Comment:
nit: I would just include this directly in the string below, since its a constant

Reply:
I thought this separation would, similar to the `clusterUrl` variable, help make the `authMechanism` value be more conspicuous until we get the feature to highlight the background of specific lines of code, and used it across the authentication snippets. If you need this change, I'll create a new ticket.